synfig-studio/src/gui/app.h

```diff
@@ -84,6 +85,9 @@ class Preferences;
 namespace studio {
 
 typedef Gtk::UIManager UIManager;
+typedef Gtk::Builder Builder;
+typedef std::list< Glib::RefPtr<Gio::SimpleAction> > MenuActionGroup;
```

Comment:
Sorry if I miss something. 
Why not use `Gio::SimpleActionGroup` instead?

Reply:
@ice0 thank you for noticing. I was unsure how to iterate through the actions with ```Gio::SimpleActionGroup```. How do you use an iterator with ```Gio::SimpleActioGroup``` to iterate through the actions for ```set_enable```?

Reply:
```
auto group = Gio::SimpleActionGroup::create();
for (const auto& action_name : group->list_actions()) {
	auto action = group->lookup_action(action_name);
	// convert Gio::Action to Gio::SimpleAction to have access to `set_enabled()` method
	if (auto simpleAction = Glib::RefPtr<Gio::SimpleAction>::cast_dynamic(action)) {
		simpleAction->set_enabled(false);
	}		
}
```